src/apps/api/views/competitions.py

```diff
@@ -337,6 +337,27 @@ def create_dump(self, request, pk=None):
         serializer = CompetitionCreationTaskStatusSerializer({"status": "Success. Competition dump is being created."})
         return Response(serializer.data, status=201)
 
+    def run_new_task_submissions(self, phase, tasks):
+        tasks_id = set([task.id for task in tasks])
+        submissions = phase.submissions.prefetch_related("children").all()
+
+        for submission in filter(lambda s: s.has_children, submissions):
+            child_tasks = set([sub.task for sub in submission.children.all()])
+            missing_tasks = tasks_id - set([task.id for task in child_tasks])
```

Comment:
```suggestion
            child_task_ids = set([sub.task.id for sub in submission.children.all()])
            missing_tasks = task_ids - child_task_ids
```
I don't see `child_tasks` being used anywhere, but in the list comprehension to get the ids. This makes we think that we can just do it in one list comprehension as shown above.